tiatoolbox/data/__init__.py

```diff
@@ -102,10 +104,12 @@ def _local_sample_path(path: pathlib.Path) -> pathlib.Path:
     """Get the path to a data file bundled with the package.
 
     Args:
-        path (pathlib.Path): Relative path to the package data file.
+        path (pathlib.Path):
+            Relative path to the package data file.
 
     Returns:
-        pathlib.Path: Path within the package to the data file.
+        pathlib.Path:
+            Path within the package to the data file.
 
 
     Example:
```

Comment:
Line 126, `Get target image for ....`

Reply:
Also add `stainnorm` to white list or rename to `stain_norm`

Reply:
> Line 126, `Get target image for ....`

I am not sure what this is referring to. The line numbers have changed. Can you please use a permalink?

Reply:
https://github.com/TissueImageAnalytics/tiatoolbox/blob/d2ed5cef96aa95482d78f129c4514bf792c59f93/tiatoolbox/data/__init__.py#L126
Update docstring to `Get target image for ...`